test/cmd/basicresources.sh

```diff
@@ -338,7 +338,6 @@ os::cmd::expect_failure_and_text "oc new-project --token="$( oc sa get-token bui
 
 # test oc projects
 os::cmd::expect_failure_and_text 'oc projects test_arg' 'no arguments'
-os::cmd::expect_success_and_text 'oc projects' 'You have access'
```

Comment:
As I mentioned in the previous comment, there is a test that does exactly that like 3 lines down